bellhop/migrations/2019-08-05-153946_sheriff/up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE sheriff (
+    primary_key BOOLEAN PRIMARY KEY NOT NULL DEFAULT true CHECK (primary_key),
```

Comment:
Diesel requires a primary key. The rest is just a hack to only ever allow one row in the table.